src/makeCalculator.js

```diff
@@ -4,7 +4,25 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add: (num) => (calculator.result += num),
+    subtract: (num) => (calculator.result -= num),
+    multiply: (num) => (calculator.result *= num),
+    divide: (num) => (calculator.result /= num),
+    reset: () => {
+      calculator.result = 0;
+
+      return calculator;
+    },
+    operate: (callback, num) => {
+      callback(num);
+
+      return calculator;
+    },
```

Comment:
[CODE KNOWLEDGE] - For the `operate` method, you should also use the method shorthand syntax. Remember to use 'this' to refer to the current object inside the method.